accounting/entries.go

```diff
@@ -0,0 +1,123 @@
+package accounting
+
+import (
+	"fmt"
+
+	"github.com/lightningnetwork/lnd/lnrpc"
+)
+
+// getPriceFunc is a function that gets the price of msat value at a given time.
+type getPriceFunc func(amtMsat, ts int64) (float64, error)
```

Comment:
do we need this declaration twice? (it's in report.go)

Reply:
No, accidentally moved it across commits.